assets/_sass/bootstrap/bootstrap/_scaffolding.scss

```diff
@@ -17,21 +17,6 @@
 }
 
 
-// Body reset
-
-html {
-  font-size: 10px;
-  -webkit-tap-highlight-color: rgba(0,0,0,0);
-}
-
-body {
-  font-family: $font-family-base;
```

Comment:
Is the default font family then used anywhere else? And if not, is this okay? I think it is, but that does mean that a default Presidium site without any styling will not use our default fonts and text colour.

Reply:
@zelrestemmet Yes used it's used as a reset https://github.com/SPANDigital/presidium-theme-website/blob/865cd881c31bc5ead5090f61589792f1d0d5be88/assets/_sass/bootstrap/bootstrap/mixins/_reset-text.scss#L2

However, looking at this again I think we can keep the body font styles for now and only remove the `html` font styles.

Reply:
Updated